pages/governance.md

```diff
@@ -0,0 +1,235 @@
+
+## Decision Making Process
+
+This section outlines how fiscal and non-fiscal decisions are made in the JuMP
+project.
+
+### Fiscal Decisions
+
+All fiscal decisions are made by the Steering Council to ensure any funds are
+spent in a manner that furthers the mission of JuMP. Fiscal decisions require
+majority approval by acting Steering Council members.
+
+The Steering Council can gain addition decision-making power if the core
+contributors decide to delegate, e.g., choosing a chair for a JuMP-dev workshop.
+
+Community members proposing decisions with a fiscal aspect should contact the
+Steering Council directly, or table their proposal as an agenda item for
+discussion on a Monthly developer call (see below).
+
+### Non-fiscal Decisions
+
+All non-fiscal decisions are made via consensus of the core contributors and
+relevant repository maintainers. Emeritus core contributors are not considered
+to be core contributors for this purpose.
+
+Code-related decisions, such when a pull request is ready to be accepted and
+merged, should be discussed via the relevant Github issues and pull requests. If
+consensus cannot be achieved, the community member proposing the change may be
+invited by a core contributor to present their proposal at a Monthly developer
+call for further discussion and community input.
+
+Non-code-related technical decisions, such long-term strategic planning for
+JuMP, should either be discussed in a Github issue, or tabled as an agenda item
+and discussed on a Monthly developer call.
+
+If consensus on a non-fiscal decision cannot be achieved, the final decision
+will be made by the BDFL.
+
+## Github Permissions
+
+For security, all core contributors and repository maintainers must have
+two-factor authentication enabled through their Github account.
+
+### Core Contributors
+
+Core contributors are added as [Owners](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization#permission-levels-for-an-organization)
+of the [JuMP-dev Github organization] and have [Admin][permissions] permission
+to every repository.
+
+### Emeritus Core Contributors
+
+Unless directed, emeritus core contributors retain the same commit rights as
+current core contributors.
+
+### Repository Maintainers
+
+Repository maintainers have [Maintain][permissions] rights to an individual
+repository as an outside collaborator. Among other rights, this allows them to
+push code to a branch in the [JuMP-dev Github organization] instead of their
+personal fork, merge pull requests, and close issues.
+
+## Community Involvement
+
+The JuMP project highly values the contributions made by members of the
+community. As an open-source project, JuMP is both made for the community, and
+by the community.
+
+There are four main channels that JuMP uses to engage with the community.
+
+### Community Forum
+
+The Steering Council will maintain a forum for community members to post
```

Comment:
The steering council isn't doing much to maintain the forum. Do we need to assign this task to them, or just say that *there is* a community form? Same for the chat room.